File: contacting-conf-web/src/main/scala/nl/ing/api/contacting/conf/domain/SurveyUpdateVO.scala
package nl.ing.api.contacting.conf.domain

/**
 *
 * @param settings
 * @param formatsAdded
 * @param formatsRemoved
 */
case class SurveyUpdateVO(settings: SurveySettingVO, formatsAdded: Seq[SurveyPhoneNumberFormatVO], formatsRemoved: Seq[SurveyPhoneNumberFormatVO])

File: contacting-conf-web/src/main/java/nl/ing/api/contacting/conf/domain/model/surveysetting/SurveyUpdateVO.java
package nl.ing.api.contacting.conf.domain.model.surveysetting;

import java.util.List;

public record SurveyUpdateVO(
        SurveySettingVO settings,
        List<SurveyPhoneNumberFormatVO> formatsAdded,
        List<SurveyPhoneNumberFormatVO> formatsRemoved
) {}
